server/wfprev-api/pom.xml

```diff
@@ -13,6 +13,15 @@
     <groupId>ca.bc.gov.nrs.wfprev</groupId>
     <artifactId>wfprev-api</artifactId>
     <version>1.0.0-SNAPSHOT</version>
+    <repositories>
+        <repository>
```

Comment:
These changes aren't needed, they can be removed

Reply:
removed